stylus/components/forms.styl

```diff
@@ -275,6 +275,71 @@ $input-text--medium
     border-radius .125rem
     padding .485rem 1rem .64rem
 
+$checkbox
+    // To make sure that checkbox's wrapper have the height of the checkbox
+    display flex
+
+    span
+        position relative
+        display inline-block
+        height checkbox-size
+        padding-left checkbox-size * 1.4
+        line-height 1rem
+        cursor pointer
+
+        &::before
+        &::after
+            content ''
+            position absolute
+            left 0
+            top 0
+            box-sizing border-box
+            width checkbox-size
+            height checkbox-size
+            border-radius .125rem
+
+        &::before
+            transition box-shadow 350ms cubic-bezier(0, .89, .44, 1)
+            box-shadow inset 0 0 0 .125rem alpha(charcoalGrey, .25)
+            background-color white
+
+            &:hover
+                box-shadow inset 0 0 0 .125rem dodgerBlue
```

Comment:
I am not able to make the hover work. I guess the hover should be put on the parent ?